core/tabs/applications-setup/tab_data.toml

```
@@ -222,6 +222,15 @@ description = "Waterfox is the privacy-focused web browser engineered to give yo
 script = "browsers/waterfox.sh"
 task_list = "FI"
 
+[[data]]
+name = "Game Engines"
+
+[[data.entries]]
+name = "Redot Engine"
+description = "Redot Engine Allows You To Make 3D Games And 2D Games And Has A Really Good OpenSource Community"
```

```suggestion
description = "Redot is a feature-packed, cross-platform game engine to create 2D and 3D games from a unified interface"
```
i think this would be better
its from redot's readme